README.md

```diff
@@ -64,29 +64,27 @@ This is not actually a runnable example, as it needs more information as specifi
 
 ```js
 videojs('video', {}, function() {
-  
+
   var player = this;
   player.ads(); // initialize the ad framework
-  
+
   // request ads whenever there's new video content
   player.on('contentupdate', function(){
     // fetch ad inventory asynchronously, then ...
     player.trigger('adsready');
   });
-  
+
   player.on('readyforpreroll', function() {
     player.ads.startLinearAdMode();
     // play your linear ad content
     player.src('http://url/to/your/ad.content');
-    player.one('durationchange', function(){
-      player.play();
-    });
-    // when all your linear ads have finished...
-    player.one('ended', function() {
+
+    // when all your linear ads have finished… do not confuse this with `ended`
+    player.one('adended', function() {
```

Comment:
Hmm... ok, it looks like `adended` gets called before `adend`, though in this implementation, it would seem that the `one()` call makes it do exactly the same thing?

Will update.